src/UICrafter.Client/Components/AppViewDesigner.razor

```diff
@@ -0,0 +1,87 @@
+@using static UICrafter.Client.Pages.UICrafterPage
+
+<MudContainer MaxWidth="MaxWidth.Small">
+    <MudDropContainer T="UIComponent" @ref="DropContainer" Items="UIComponents" ItemsSelector="@((item,dropzone) => item.DropZoneID == dropzone)" ItemDropped="ComponentUpdated" Class="d-flex flex-wrap flex-grow-1">
+        <ChildContent>
+            <MudDropZone T="UIComponent" Identifier="Drop Zone 1" AllowReorder="true" Class="rounded mud-background-gray pa-2 flex-grow-1" Style="height: 45rem; overflow: auto;">
+            </MudDropZone>
+        </ChildContent>
+        <ItemRenderer>
+            @if (context is UIButton button)
```

Comment:
As this might grow large, It could be considered to be a seperate component. But if You do you should figure a @bind interface that makes sense for all of the different values. I cant quite figure out if this would be simpler or if it would be worse. Just an Idea. 😄

Reply:
Har tænkt lidt på det, bliver ikke lige top prioritet